### ISSUE TITLE
Bulk add all indicators

### ISSUE THREAD

**Issue by dorienh** [mrjbq7/ta-lib#396]
Is there a quick way to add all indicators as columns to my pandas dataframe / or equivalent OHLV numpy without having to call them individually?

**Comment by mrjbq7** [comment]
What parameters would you use?

**Comment by dorienh** [comment]
I suppose there are no default settings. That makes sense, but I'm just
trying to quickly bulk add TIs for a machine learning project.

Would I need  to delve up each indicator function and check what type of
settings they need?

**Comment by luoyikang** [comment]
Yes. As for a deep learning network, I don`t wish pick up some useful features one by one but aggregate all features as an input. It would be pretty convenient if there is one API that can return all features. Anyway, it`s a small suggestion for this project.
